src/components/calcite-tooltip/calcite-tooltip.scss

```diff
@@ -2,16 +2,22 @@
 @include popperArrow();
 
 .container {
-  position: relative;
-  background: var(--calcite-ui-foreground-1);
-  max-width: 300px;
-  max-height: 300px;
-  display: flex;
-  justify-content: flex-start;
-  flex-direction: column;
-  font-weight: 500;
-  color: var(--calcite-ui-text-1);
-  padding: 12px 16px;
-  overflow: hidden;
+  @apply relative 
+    bg-foreground-1 
+    flex 
+    justify-start 
+    flex-col 
+    font-medium
+    text-color-1 
+    py-3 
+    px-4 
+    overflow-hidden 
+    text--1-wrap
+    border
+    border-solid
+    border-color-3;
   @include font-size(-3);
+  max-width: 20rem;
+  max-height: 20rem;
+  border-radius: var(--calcite-border-radius);
```

Comment:
I thought we decided we would use the [defaults](https://v1.tailwindcss.com/docs/border-radius) for `border-radius`? 

We could set vars for those values if we must.. but either way, we should be using a util here.